README.capture-agent-manager.md

```diff
@@ -0,0 +1,92 @@
+# enabling the capture agent manager app
+
+the capture-agent-manager app is meant to be deployed in the utilities layer of
+a mh cluster.
+
+the capture-agent-manager app is a flask-gunicorn web app to keep capture agent
+inventory, and provide an api/ui for configuring capture agents, as well as
+switching from primary to secondary live stream..
+
+* modify your cluster config to include the following layer config:
+
+```
+      {
+        "name": "Utility",
+        "shortname": "utility",
+        "type": "custom",
+        "enable_auto_healing": true,
+        "install_updates_on_boot": true,
+        "use_ebs_optimized_instances": true,
+        "auto_assign_elastic_ips": true,
+        "auto_assign_public_ips": true,
+        "custom_recipes": {
+          "setup": [
+            "mh-opsworks-recipes::set-timezone",
+            "mh-opsworks-recipes::fix-raid-mapping",
+            "mh-opsworks-recipes::set-bash-as-default-shell",
+            "mh-opsworks-recipes::install-utils",
+            "mh-opsworks-recipes::enable-postfix-smarthost",
+            "mh-opsworks-recipes::install-custom-metrics",
+            "mh-opsworks-recipes::create-alerts-from-opsworks-metrics",
+            "mh-opsworks-recipes::enable-enhanced-networking",
+            "mh-opsworks-recipes::clean-up-package-cache",
+            "mh-opsworks-recipes::create-capture-agent-manager-user",
+            "mh-opsworks-recipes::create-capture-agent-manager-directories",
+            "mh-opsworks-recipes::install-capture-agent-manager-packages",
+            "mh-opsworks-recipes::install-capture-agent-manager",
+            "mh-opsworks-recipes::configure-capture-agent-manager-gunicorn",
+            "mh-opsworks-recipes::configure-capture-agent-manager-nginx-proxy",
+            "mh-opsworks-recipes::configure-capture-agent-manager-supervisor"
+          ],
+          "shutdown": [
+            "mh-opsworks-recipes::remove-alarms"
+          ]
+        },
+        "volume_configurations": [
+
+        ],
+        "instances": {
+          "number_of_instances": 1,
+          "instance_type": "t2.medium",
+          "root_device_type": "ebs"
+        }
+      },
+```
+
+* update your "custom_json" block to provide the following info to `cadash`:
```

Comment:
You might want to be clear that there are default values for all of these in `mh-opsworks-recipes` `libraries/default.rb` and indicate which ones need to be overridden.